platform/mellanox/rules.mk

```diff
@@ -31,3 +31,5 @@ $(SYNCD)_RDEPENDS += $(MLNX_SAI)
 
 # Inject mlnx sdk libs to platform monitor
 $(DOCKER_PLATFORM_MONITOR)_DEPENDS += $(APPLIBS) $(SX_COMPLIB) $(SXD_LIBS) $(SX_GEN_UTILS) $(PYTHON_SDK_API) $(APPLIBS_DEV) $(SX_COMPLIB_DEV) $(SXD_LIBS_DEV) $(SX_GEN_UTILS_DEV)
+
+export SONIC_BUFFER_MODEL=dynamic
```

Comment:
Do not change build time config in component makefile, if you need, change the pipeline command line.

Reply:
Hi Qi,
Yes, it works to change the CLI option but what's the benefit to change a working code?